src_tests/unittests/osm_configurator/view/popups/test_yes_no_pop_up.py

```diff
@@ -0,0 +1,20 @@
+import os.path
+from src_tests.definitions import TEST_DIR
+
+from pathlib import Path
+import src.osm_configurator.view.popups.yes_no_pop_up as ynpu
+
+
+def _prepare(target_path):
+    result_folder = os.path.dirname(target_path)
+    if not os.path.exists(result_folder):
+        os.makedirs(result_folder)
+    if os.path.exists(target_path):
+        os.remove(target_path)
+
+
+@pytest.mark.skip(reason="pytest not supported yet")
+@pytest.mark.parametrize(["Ich Kek", "Du Kek", "Er sie es Kek", "Wir Kek", "Ihr Kek", "Sie Kek"])
```

Comment:
explenation needed for this "weird" list

Reply:
The first decorator will skip the test if pytest is executed. The second decorator will execute the test with the given parameteres multiple times.
However, pytest is not implemented yet (in this branch at least). Therefore, to make it work correctly we'll need to merge the current dev-branch into view to implement the needed config files to execute pytest.